framework/src/source/BaseTestSuite.bs

```diff
@@ -2260,6 +2260,36 @@ namespace rooibos
       end for
       m.mocks = invalid
       rooibos.resetMocksByFunctionName()
+
+      ' Clean up the global functions mocks as well
+      globalAa = getGlobalAa()
+      globalAa._globalMocks = invalid
+    end function
+
+    ' /**
+    '  * @memberof module:BaseTestSuite
+    '  * @name mockGlobalFunction
+    '  * @function
+    '  * @instance
+    '  * @description Adds a function to the global mock for the current components scope
+    '  * @param {String|Function} functionOrFunctionName - a function by reference or the string name of a function to mock
+    '  * @param {Function} mockFunction - the function to use as a mock
+    '  */
+    function mockGlobalFunction(functionOrFunctionName as dynamic, mockFunction as function) as void
```

Comment:
I don't think we need a new method here, as we discussed. will expand on that in the review.

Reply:
Removed and consolidated to `m.stubCall()`